test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -54,6 +55,18 @@ public Steps()
             _random = new Random();
         }
 
+        public async Task ThenConfigShouldBe(FileConfiguration fileConfig)
+        {
+            var internalConfigCreator = _ocelotServer.Host.Services.GetService<IInternalConfigurationCreator>();
+
+            var internalConfigRepo = _ocelotServer.Host.Services.GetService<IInternalConfigurationRepository>();
+            var internalConfig = internalConfigRepo.Get();
+
+            var config = await internalConfigCreator.Create(fileConfig);
+
+            internalConfig.Data.RequestId.ShouldBe(config.Data.RequestId);
```

Comment:
I don't like this. But shouldly doesn't implement an ShouldBeEquivalentTo method as FluentAssertions does. Do you have any suggestion?

Reply:
Dont worry this is fine, Yeh FluentAssertions has done well adding that feature...maybe I will move to them one day!